base/uk.ac.stfc.isis.ibex.ui/src/uk/ac/stfc/isis/ibex/ui/tables/DataboundTable.java

```diff
@@ -84,8 +84,9 @@
      * @param parent the parent
      * @param style the style
      * @param tableStyle the table style
+     * @param emptyRow to add empty row or not
      */
-    public DataboundTable(Composite parent, int style, int tableStyle) {
+    public DataboundTable(Composite parent, int style, int tableStyle, boolean emptyRow) {
```

Comment:
rather than specifying `emptyRow` to `false` on every table that inherits from `DataboundTable` is it possible to override a constructor that just specifies `parent, style, tablestyle`? This means that you can just call this constructor like so: 

```java
public DataboundTable(Composite parent, int style, int tableStyle) {
    this(parent, style, tableStyle, false);
}
```

and leave the rest of the tables that aren't to do with the script generator unmodified.